spec/components/error_message_spec.rb

```diff
@@ -8,13 +8,13 @@ def component_name
   it "renders error message" do
     render_component(text: "Please enter your National Insurance number")
 
-    assert_select(".govuk-error-message", text: "Error: Please enter your National Insurance number")
+    assert_select("p.govuk-error-message", text: "Error: Please enter your National Insurance number")
```

Comment:
@andysellick 

---

I see why you've modified this test but I don't think we need to specifically test the correct HTML element is used here - otherwise it would have previously looked for a `span`.

Reply:
In the tests we use for other components, such as the checkboxes code above we test for the element type as well, so I've added this for a bit of consistency, and to make sure we stick to using paragraphs instead of spans.